### DISCUSSION TITLE
Duplicate "link" icon in Draftail Editor and external links turned to normal text after refresh

### DISCUSSION THREAD

**Discussion by joelting92** [wagtail/wagtail#12105] [General]
Hi, I just would like to ask if anyone know what is happening with my Draftail Editor? It has 2 links icons, which is not the case in a fresh install and whenever I input an external link, after I save as draft, it will turn into normal text after reloading. The publish live page is showing the link without an issue though. If I republish it again, then it will turn into normal text. I'll attach screenshot as below:

![image](https://github.com/wagtail/wagtail/assets/26118121/1e260433-fcff-4aa6-9aff-3b08afac3f37)

Before I save as draft:
![image](https://github.com/wagtail/wagtail/assets/26118121/a05e410a-cc54-4bc3-97b1-f6f25c82c538)

After I save as draft or refresh the page:
![image](https://github.com/wagtail/wagtail/assets/26118121/bf7ecf73-37a1-4673-8786-150fa44a17ea)

I'm not sure if the 2 issues are related. I'm not able to replicate this in a fresh wagtail site. I did not do any customization to the Draftail Editor. Any insights is really appreciated.

**Comment by RealOrangeOne** [comment]
Have you defined `feature` at all on the richtext block or in your settings? Duplicates there could show up like this.

**Comment by thibaudcolas** [comment]
The "anchor" icon in the toolbar isn’t a built-in feature of Wagtail, so I’d suspect there’s an issue in the implementation of this customization.

**Reply by joelting92** [reply]
> The "anchor" icon in the toolbar isn’t a built-in feature of Wagtail, so I’d suspect there’s an issue in the implementation of this customization.

You are right... I just noticed that, probably there's some customization that I am not aware of. Appreciate the help. I'll take a look. Thanks for pointing me to the right direction.

**Reply by joelting92** [reply]
> The "anchor" icon in the toolbar isn’t a built-in feature of Wagtail, so I’d suspect there’s an issue in the implementation of this customization.

You are a lifesaver! I found out the issue, relating to a package that was installed. Thank you so much!
